pyaedt/edb_core/stackup.py

```diff
@@ -218,6 +219,47 @@ def create_debye_material(
         )
         return self._add_dielectric_material_model(name, material_def)
 
+    @aedt_exception_handler
+    def flip_stackup(self):
+        """Flip the current layer stackup of a layout.
+        Parameters
+        ----------
+        Returns
+        -------
+        type bool
+            True when layer stackup update success False if not.
+        """
+        lc = self._active_layout.GetLayerCollection()
+        new_lc = self._edb.Cell.LayerCollection()
+        max_elevation = 0.0
+        for lyr in lc.Layers(self._edb.Cell.LayerTypeSet.StackupLayerSet):
+            if not 'RadBox' in lyr.GetName():  # Ignore RadBox
+                lower_elevation = lyr.GetLowerElevation() * 1.0e6
+                upper_elevation = lyr.GetUpperElevation() * 1.0e6
+                max_elevation = max([max_elevation, lower_elevation, upper_elevation])
+
+        non_stackup_layers = []
+        for lyr in lc.Layers(self._edb.Cell.LayerTypeSet.AllLayerSet):
+            cloned_layer = lyr.Clone()
+            if not cloned_layer.IsStackupLayer():
+                non_stackup_layers.append(cloned_layer)
+                continue
+
+            if not 'RadBox' in lyr.GetName():
+                upper_elevation = lyr.GetUpperElevation() * 1.0e6
+                updated_lower_el = max_elevation - upper_elevation
+                val = self._edb_value("{}um".format(updated_lower_el))
+                cloned_layer.SetLowerElevation(val)
+                new_lc.AddStackupLayerAtElevation(cloned_layer)
+
+        layer_list = general.convert_py_list_to_net_list(non_stackup_layers)
+        new_lc.AddLayers(layer_list)
+        if self._active_layout.SetLayerCollection(new_lc):
```

Comment:
I have a question. 
You have excluded the RadBox from the flipping algorithm, but then it is not added back to the ``new_lc`` layer.
Is that correct?

Reply:
that's correct since we build a new stackup and override the original one